data/bots/lib/libbot/geo-features.ts

```diff
@@ -8,14 +10,33 @@ export class GeoFeatures {
     this.baseHttpClient = httpClient.baseHttpClient;
   }
 
-  async getIdFromQueryStringInProject(
+  async getIdOfFeaturesBySubstringMatchOnName(
     projectId: string,
     name: string,
-  ) {
-    return await this.baseHttpClient.get(
+  ): Promise<string[]> {
+    const opStart = Process.hrtime();
```

Comment:
we may want to consider adding pretty decorator for those (like https://github.com/norbornen/execution-time-decorator/blob/master/src/index.ts implementation or package directly) to avoid bloating code functions

Reply:
Yes, I have been postponing this, but inline hrtime diffs as a rule rather than as a one-off is ugly.

I have created a technical-debt task as I won't be able to take care of this while I try to close outstanding stuff before the break: https://vizzuality.atlassian.net/browse/MARXAN-1138